### PR TITLE
Leaflet update

### PR DESCRIPTION
**Pull request by lightonflux**
Update leaflet from 0.7.3 to 0.7.7. That's only bug and security fixes, and no new features.

### CODE REVIEW

**Comment by est31** [comment]
Nice!

Can you git rebase -i to remove the second commit? It seems to be not required, no?

File: www/leaflet.css [hunk 1]
```diff
@@ -21,6 +21,7 @@
 .leaflet-container {
 	overflow: hidden;
 	-ms-touch-action: none;
+	touch-action: none;
```
Comment (est31) [inline www/leaflet.css:24]: Can you describe in the commit message what this change is there for?

Best use a format where the first line is concise, the second line empty, and the third line contains the longer description.

**Comment by lightonflux** [comment]
The second commit is there because it makes it easy to compare older and newer versions. But just for running it is not needed. You can't compare the versions with the minified files because most diff tools can't handle it.

**Comment by lightonflux** [comment]
So the rebase worked, but then i wasn't carefull enough and now i reapplied the stuff from remote origin.

Can you tell me how to remove the doubled and unecessary commits?
( e0d95fe8134f7cababe90a0fe28f58853752e683, 2fb90d55ee6f32ab24ce82d9ad8b7a582003b58e, 55089c48f42127af87f133629b30e2ae584c6a88)

https://github.com/lightonflux/leaftest/commits/leaflet-update

When i try 

```
 git rebase -i b0ce62becba0661a4912be02f1cc738c7b67daee
```

I get the "empty warning" and git recommend     `git commit --allow-empty`

**Comment by est31** [comment]
@lightonflux do git rebase -i then you can tell git whether to drop or pick the commits.

**Comment by est31** [comment]
I think your mistake is that you rebased on the wrong commit, rebase on `5c56b38b8977cac8932429c185665370cd561f1d` instead.

**Comment by lightonflux** [comment]
Wasn't able to redo rebase it correctly and upload to github (it always rewrote local from remote). So closing here and doing a new PR.

**Pull request closed without merging**